talk/sphinx.slide

```diff
@@ -0,0 +1,78 @@
+Interfaces will Save the Future
+Case Study: Sphinx Rate Limiter
+
+21 Jan 2015
+
+
+Mohit Gupta & Alex Zylman
+@mohitgupta
+@amzylman
+
+Clever
+https://clever.com
+https://github.com/Clever
+
+
+# short introductions: name and position + 2 line summary of Clever
+
+
+* But what is Sphinx?
+
+
+- HTTP rate limiter
+- Based on leakybucket algorithm (supporting both Redis and in-memory stores)
+- Supports matching on request headers and IP address
+- Transparent limiting through X-Rate-Limit headers
+- Low request latencies (~15 ms)
+
+.image logo.png _ 250
+
+
+* A request in Sphinx-land
+
+
+* ♥ Interface ♥
+
+- Better collaboration
```

Comment:
I would consider the sentence "Repo A has better collaboration than repo B" to have awkward phrasing. Consider "Easier to collaborate" or, to mach the title of the next slide "Improved collaboration"